src/deploy_inst.cc

```diff
@@ -3,7 +3,11 @@
 
 namespace cycamore {
 
-DeployInst::DeployInst(cyclus::Context* ctx) : cyclus::Institution(ctx) {}
+DeployInst::DeployInst(cyclus::Context* ctx)
+    : cyclus::Institution(ctx),
```

Comment:
I feel like the toolkit addition would be less cumbersome if it were inherited here, rather than copy-pasted into each agent. Thoughts?

Reply:
I'm sorry, my knowledge on C++ is a little outdated here so I may not be understanding this properly. 

Do you mean that it would be less cumbersome if:
- I had inherited the Position toolkit in the DeployInst class? (since agents under DeployInst would also inherit the toolkit?)

Reply:
@katyhuff is suggesting that DeployInst (and others) have an "is-a" relationship with Position (a DeployInst is a Position), rather than what is here which is "has-a" (a DeployInst has a Position).  Semantically, I think that "has-a" relationships are nicer, but the way you have written it here, an is-a relationship would be less typing. So I don't mind either way :smile:

Reply:
Please ping again when you have updated this, since it will be a pretty invasive change compared to what you have now. Thanks!

Reply:
@scopatz is of course correct. I remember we went back and forth about this in my office, @gyutaepark and arrived at has-a. Seeing it now, with the extent of the boilerplate text needed, made me think we might have gone the wrong direction with the has-a relationship. In any case, I agree with @scopatz that it's fine as is, but perhaps if it's cumbersome in the future, we can keep in mind a future implementation as an is-a instead.

Reply:
(which is all to say that if @scopatz is ok with has-a, I'm ok with it too.) No need to reimplement. Let's just keep in mind the potential for simplification, in case it's too burdensome for some reason in the future.

Reply:
I remember now. I'll keep this in mind.
Thank you for pointing it out. I can certainly change the implementation once I have completed my work on ArcGIS